modules/core/src/main/scala/fs2/kafka/ConsumerSettings.scala

```diff
@@ -417,6 +425,15 @@ object ConsumerSettings {
     override val recordMetadata: ConsumerRecord[K, V] => String,
     override val maxPrefetchBatches: Int
   ) extends ConsumerSettings[F, K, V] {
+
+    def withPlainSasl(usernameToken: String, passwordToken: String): ConsumerSettings[F, K, V] =
```

Comment:
I've never worked with SASL but from what I can see here in `PlainLoginModule` there's no reference to any token.

Why not removing that prefix? 👇🏽 

```suggestion
    def withPlainSasl(username: String, password: String): ConsumerSettings[F, K, V] =
```